compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.uikit.kt

```diff
@@ -273,6 +274,15 @@ internal class ComposeSceneMediator(
         renderingUIViewFactory(interopContext, renderDelegate)
     }
 
+    private val applicationStateListener = ApplicationStateListener { isForeground ->
+        // Sometimes the application can trigger animation and go background before the animation is
+        // finished. The scheduled GPU work is performed, but no presentation can be done, causing
+        // mismatch between visual state and application state. This can be fixed by forcing
+        // a redraw when app returns to foreground, which will ensure that the visual state is in
+        // sync with the application state even if such sequence of events took a place.
+        onComposeSceneInvalidate()
```

Comment:
it's supposed to be only scene callback. Please call `renderingView.needRedraw()` instead

Reply:
Done